setup.py

```diff
@@ -23,6 +23,9 @@
 
 INSTALL_REQUIRES = [
```

Comment:
> Or was it perhaps intentional, so that developers must remember to pick a version of Cylc to build the documentation against?

Yep! It's entirely down to the developer!